server/controllers/payroll/reports/payslipGenerator.js

```diff
@@ -30,11 +30,20 @@ const DEFAULT_OPTS = {
 function build(req, res, next) {
   const options = _.clone(req.query);
 
+  if (Array.isArray(options.employees)) {
```

Comment:
One way you might simplify this code up is to make `options.employees` always an array.  You can do this with [`Array.prototype.concat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Concat).  Like this:
```js
options.employees = [].concat(options.employees); // now options.employees is an array
```
Then you can simply use a `map()` to process the uuids to binary form.
```js
options.employees = options.employees.map(uid => db.bid(uid));
```